test/generators/component_generator_test.rb

```diff
@@ -7,6 +7,7 @@ class ComponentGeneratorTest < Rails::Generators::TestCase
   setup :prepare_destination
 
   test "Assert all files are properly created" do
+    skip
```

Comment:
We can always re-run them. I think it's preferable to have the tests as false negatives and re-run them than to skip them.

I tried figuring out why it randomly failed but wasn't successful. Any feedback is welcome on that!